src/scripts/run-bats.sh

```diff
@@ -15,5 +15,5 @@ if [ -n "$ORB_VAL_ARGS" ]; then
 fi
 
 set -x
-bats --formatter "$ORB_VAL_FORMATTER" "$@" "$ORB_VAL_PATH" | tee "$ORB_VAL_OUTPUT"
+bats --formatter "$ORB_VAL_FORMATTER" --report-formatter "$ORB_VAL_REPORT_FORMATTER" "$@" --output "$ORB_VAL_OUTPUT" "$ORB_VAL_PATH" 
```

Comment:
I'm not convinced that we should be wrapping `$ORB_VAL_PATH` in quotes.

In the previous code, users could specify as many paths as they wanted as long as they separated them with whitespace (and wrapped any paths that contained spaced within quotes).
With the code this way, user's _can't_ specify more than one path ... and if they wrap things in quotes now then those quotes will be preserved and might cause surprising behavior.

I accept that this isn't a clear-cut issue (e.g. if the parameter was called "paths", plural, or if CircleCI allowed the passing of arrays, then that'd clinch the vote ... and normally I'm 100% in favour of wrapping variable expressions in quotes to preserve spaces as they're very common on e.g. Windows builds) but in this case I would be wary of adding quotes because that's actually a reduction in functionality (you can't specify multiple paths anymore).

Note: Everything else needs quotes; everything else is always a single parameter; it's only ORB_VAL_PATH that's difficult.